backend/app/emails.py

```diff
@@ -29,6 +29,8 @@
 
 class EmailCategory(str, Enum):
     SECURITY_LOGIN = "security_login"
+    DEVELOPER_INVITE = "developer_invite"
+    DEVELOPER_INVITE_ACCEPTED = "developer_invite_accepted"
```

Comment:
Makes me wonder, if we should send the inviting party an email, when the invitee declines.